contrib/labstack/echo.v4/echotrace_test.go

```diff
@@ -163,7 +167,6 @@ func TestError(t *testing.T) {
 		called = true
 
 		err := wantErr
-		c.Error(err)
```

Comment:
@rarguelloF I've updated the test to ensure that the error handler is called once, and I've removed the `c.Error` call here as well, since this is a test handler and from the [echo docs](https://github.com/labstack/echo/blob/447c92d842e2f91c07eb032f619bea212beeee60/context.go#L178) we don't want to explicitly invoke the `c.Error` inside of handlers.

```// Avoid using this method in handlers as no middleware will be able to effectively handle errors after that.```

Reply:
The fact that this change has removed this call and not broken anything in the rest of test should hopefully further highlight the fact that the expected behavior is already happening without this call, and explicitly calling it is causing as a second, unnecessary call.